packages/atomic/stencil.config.ts

```diff
@@ -84,6 +85,7 @@ export const config: Config = {
     reloadStrategy: 'pageReload',
   },
   plugins: [
+    // https://github.com/fabriciomendonca/stencil-inline-svg/issues/16
```

Comment:
another warning we had, issue was already opened, might keep it here a bit and then just make a PR there if nothing is done